test/cli/knexfile-test.spec.js

```diff
@@ -65,6 +65,33 @@ module.exports = {
         );
       });
 
+      it('Run migrations with knexfile returning function passed', () => {
```

Comment:
Can you also add same tests for seeder execution? Ideally the two should work symmetrically.